src/classLibrary/Utility/GuidGenerator.ts

```diff
@@ -0,0 +1,39 @@
+import { Core } from "@kirinnee/core";
+import { IGuidGenerator } from '../interfaces/interfaces';
+
+class GuidGenerator implements IGuidGenerator {
+
+    private static guids: string[] = [];
+    public readonly c: Core;
+
+    constructor(core: Core) {
+        core.AssertExtend();
+        this.c = core;
+    }
```

Comment:
Actually, would it be better to put this under the Utility class? I'm not sure 😅 I just want to know the reason for creating a separate class for this instead of putting the functions under the `Utility` class. 😁

Reply:
oh i just put it separately cuz i was thinking the `GuidGenerator` most likely just used by `GuidResolver` and its generating something and is diff from lets say `FlattenObject` , `Throw` which is much more generic and is like a helper function kind of thing but yeah tbh its just my intuition lol